Makefile

```diff
@@ -161,7 +164,7 @@ CCFLAGS += 			\
 	-nostdlib       \
 	-mlongcalls	\
 	-mtext-section-literals
-#	-Wall			
```

Comment:
My editor removes trailing whitespace. I figure we'd rather not have trailing whitespace so I didn't put it back.